src/components/common/PeopleAndRoles/CurrentOfficers.vue

```diff
@@ -0,0 +1,314 @@
+                  </span>
+                  <span
+                    v-else
+                    class="officer-change-text"
+                  >
+                    Add Officers
+                  </span>
+                </a>
+              </span>
+            </template>
+            <span>To manage officers, submit or discard any other changes to business information.</span>
+          </v-tooltip>
+        </template>
+
+        <template v-else>
+          <a
+            :href="officerChangeUrl"
+            class="officers-change"
+            @click="handleClick"
+          >
+            <v-icon
+              medium
+              class="add-officer-icon"
+              color="appDkBlue"
+              v-text="officers.length > 0 ? 'mdi-pencil-outline' : 'mdi-account-plus-outline'"
+            />
+            <span
+              v-if="officers.length > 0"
+              class="officer-change-text"
+            >
+              Manage Officers
+            </span>
+            <span
+              v-else
+              class="officer-change-text"
+            >
+              Add Officers
+            </span>
+          </a>
+        </template>
+      </article>
+
+      <v-simple-table class="officers-table section-container">
+        <!-- List Display Section -->
+        <thead v-if="officers.length > 0">
+          <!-- List Headers -->
+          <tr class="officers-list-header pb-3">
+            <th
+              v-for="(title, index) in tableHeaders"
+              :key="index"
+              class="px-0"
+            >
+              <span class="officers-title">{{ title }}</span>
+            </th>
+          </tr>
+        </thead>
+        <thead
+          v-else
+          class="no-officers-header"
+        >
+          There are currently no officers. You can add officers if you are required to do so.
+        </thead>
+        <tbody v-if="officers.length > 0">
+          <!-- List Content -->
+          <tr
+            v-for="(orgPerson, index) in officers"
+            :key="`officer.officer.id:${index}`"
+            class="officers-content py-3"
+          >
+            <!-- Name + Badge -->
+            <td class="text-truncate px-0">
+              <!-- provide tooltip to display full name if name is longer than 25 chars -->
+              <v-tooltip
+                top
+                :disabled="formatFullName(orgPerson.officer).length < 25"
+                color="primary"
+              >
+                <template #activator="{ on }">
+                  <span
+                    class="officers-name"
+                    v-on="on"
+                  >{{ formatFullName(orgPerson.officer) }}</span>
+                </template>
+                <span class="officers-name">{{ formatFullName(orgPerson.officer) }}</span>
+              </v-tooltip>
+            </td>
+            <!-- Roles -->
+            <td class="px-0">
+              <span v-if="orgPerson.roles.length > 0">
```

Comment:
You might not need this check. I think v-for doesn't iterate at all if array length is 0.